tests/apichecks/crds_test.go

```diff
@@ -96,6 +96,11 @@ func TestMissingRefs(t *testing.T) {
 					isRef = true
 				}
 
+				if strings.HasSuffix(fieldPath, "erviceAccount") {
```

Comment:
Do all "service account" fields have to be GSA? Maybe only enforcing "googleServiceAccount" field to be reference ?

Reply:
From what I see, many GSA has name `serviceAccount` without google, some has prefix like `XXXServiceAccount`